README.md

```diff
@@ -4,11 +4,107 @@
+
+To make use of the Access Operator, create a `KafkaAccess` custom resource (CR).
+You must specify the name of the name of the `Kafka` CR you want to connect to.
+You can optionally also specify the name of the listener in the `Kafka` CR and a `KafkaUser`.
+See the [examples folder](https://github.com/strimzi/kafka-access-operator/tree/main/packaging/examples) for some valid `KafkaAccess` specifications.
+
+If you do not specify which listener you want to connect to, the operator uses the following rules to choose a listener:
+1. If there is only one listener configured in the `Kafka` CR, that listener is chosen.
+2. If there are multiple listeners listed in the `Kafka` CR, the operator filters the list by comparing the `tls` and `authentication` properties in the `Kafka` and `KafkaUser` CRs to select a listener with the appropriate security.
+3. If there are multiple listeners with appropriate security, the operator chooses the one that is of type `internal`.
+4. If there are multiple internal listeners with appropriate security, the operator sorts the listeners alphabetically by name, and chooses the first one.
+
+Once the Access Operator has created the binding `Secret`, it updates the `KafkaAccess` custom resource to put the name of the secret in the status, for example:
+
+```yaml
+...
+status:
+  binding:
+    name: kafka-binding
+```
+
+The `Secret` created by the Access Operator has the following structure:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: kafka-binding
+type: servicebinding.io/kafka
+data:
+    type: kafka
+    provider: strimzi
+
+    bootstrap.servers: # comma separated list of host:port for Kafka
+    bootstrap-servers: # comma separated list of host:port for Kafka
+    bootstrapServers: # comma separated list of host:port for Kafka
+
+    security.protocol: # one of PLAINTEXT, SASL_PLAINTEXT, SASL_SSL or SSL
+    securityProtocol: # one of PLAINTEXT, SASL_PLAINTEXT, SASL_SSL or SSL
+
+    # Provided if TLS enabled:
+    ssl.truststore.crt: #  Strimzi cluster CA certificate
+
+    # Provided if selected user is SCRAM auth:
+    username: # SCRAM username
+    password: # SCRAM password
+    sasl.jaas.config: # sasl jaas config string for use by Java applications
+    sasl.mechanism: SCRAM-SHA-512
+    saslMechanism: SCRAM-SHA-512
+
+    # Provided if selected user is mTLS:
+    ssl.keystore.crt: # certificate for the consuming client signed by the clients' CA
+    ssl.keystore.key: # private key for the consuming client
+```
+
+Developers can make this `Secret` available to their applications themselves, or use an operator that implements the [Service Binding specification](https://servicebinding.io/spec/core/1.0.0/) to do it.
+
+## Getting help
+
+If you encounter any issues while using the Access Operator, you can get help through the following methods:
+
+- [Strimzi Users mailing list](https://lists.cncf.io/g/cncf-strimzi-users/topics)
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
```

Comment:
You should also link to GitHub discussions here: https://github.com/orgs/strimzi/discussions